extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -83,23 +102,39 @@ export class LightboxManager {
    */
   scanLightboxables_() {
     return this.ampdoc_.whenReady().then(() => {
+      const viewer = elementByTag(this.ampdoc_.getRootNode(), VIEWER_TAG).id;
       const matches = this.ampdoc_.getRootNode().querySelectorAll('[lightbox]');
-      this.elements_ = [];
       for (let i = 0; i < matches.length; i++) {
         const element = matches[i];
-        if (element.getAttribute('lightbox').toLowerCase() != 'none') {
-          this.elements_.push(element);
+        if (element.tagName == 'AMP-CAROUSEL') {
+          // TODO: special case carousel
+        } else {
+          const lightboxGroupId = element.getAttribute('lightbox');
+          if (lightboxGroupId == '' || lightboxGroupId == 'default') {
+            this.lightboxGroups_.default.push(dev().assertElement(element));
+          } else {
+            if (!this.lightboxGroups_.hasOwnProperty(lightboxGroupId)) {
```

Comment:
If you make `this.lightboxGroups_` a [`src/object.js#map`](https://github.com/ampproject/amphtml/blob/master/src/utils/object.js#L30) you can use `if (!this.lightboxGroups_[lightboxGroupId]) { }` instead of `hasOwnProperty`.

If you do still need `hasOwnProperty` you should use [`src/object.js#hasOwn`](https://github.com/ampproject/amphtml/blob/master/src/utils/object.js#L63) instead